wp-content/themes/missilethreat/assets/_scss/layout/_header.scss

```diff
@@ -1 +1,289 @@
+    order: 3;
+  }
+
+  &__search {
+    position: absolute;
+    right: rem(12px);
+    left: rem(12px);
+    display: flex;
+    height: rem(48px);
+    padding: rem(12px);
+    background: $color-primary-blue-400;
+    visibility: hidden;
+    opacity: 0;
+    transition: opacity 0.3s ease-in-out, visibility 0.3s ease-in-out;
+    pointer-events: none;
+
+    &.visible {
+      z-index: 15;
+      visibility: visible;
+      opacity: 1;
+      pointer-events: all;
+    }
+  }
+
+  &__search-icon {
+    z-index: 15;
+    justify-self: right;
+    order: 3;
+    color: $color-primary-light-blue;
+    cursor: pointer;
+
+    &.invisible {
+      display: none;
+    }
+  }
+
+  &__search-label {
+    display: none;
+  }
+
+  &__search-field {
+    @extend %font-ui-text-lg;
+    width: 100%;
+    color: $color-text-light-155;
+    background: $color-primary-blue-400;
+    border: 0;
+    border-bottom: 1px solid $color-primary-blue-900;
+  }
+
+  &__search-icon,
+  &__search-submit {
+    background: none;
+    border: 0;
+  }
+
+  &__search-submit {
+    margin-right: rem(48px);
+    margin-left: rem(8px);
+    color: $color-line-mid-gray;
+    line-height: 50%;
+    cursor: pointer;
+  }
+
+  &__search-close {
+    position: absolute;
+    right: 0;
+    width: rem(48px);
+    height: rem(48px);
+    margin-top: rem(-12px);
+    margin-left: rem(20px);
+    padding: 0;
+    color: $color-primary-light-blue;
+    font-size: rem(18px);
+    line-height: 50%;
+    background: $color-primary-blue-200;
+    border: 0;
+    cursor: pointer;
+  }
+
+  @include breakpoint('small') {
+    justify-content: space-between;
+  }
+
+  @include breakpoint('large') {
+    flex-wrap: nowrap;
+    justify-content: initial;
+    padding-right: 6vw;
+    padding-left: 6vw;
+    column-gap: unset;
+    
+    &__logo {
+      order: 1;
+      justify-self: start;
+      padding-right: rem(48px);
+
+      &.invisible {
+        display: block;
+      }
+    }
+
+    .site-nav {
+      z-index: 15;
+      order: 2;
+      width: 100%;
+      margin: 0;
+    }
+
+    .primary-menu {
+      @include large-breakpoint-font-ui-text-md-semibold();
+      top: 0;
+      flex-direction: row;
+      justify-content: flex-end;
+      width: 100%;
+      white-space: nowrap;
+      @include full-width-background(0);
+      background: 0;
+      visibility: visible;
+      opacity: 1;
+      clip-path: inset(0);
+
+      &::before,
+      &::after {
+        content: none;
+      }
+
+      li {
+        padding-left: rem(8px);
+      }
+    }
+
+    .hamburger,
+    .nav-close {
+      display: none;
+    }
+
+    &__search-icon {
+      justify-self: start;
+    }
+    
+    &__search {
+      order: 3;
+      width: calc((100% - 6vw) - (6vw + 248px));
+      transition: opacity 0.3s linear;
+      
+      &.visible {
+        right: 6vw;
+        left: initial;
+      }
+    }
+  }
+
+  @include breakpoint('xxlarge') {
+    letter-spacing: 0.01em;
+  }
+}
+
+.solid-bg {
```

Comment:
Instead of `solid-bg` as a class name, I would recommend using the modifier part of the BEM naming methodology and making this something like `.header--solid-bg`. That ties it very explicitly to the header, where as `solid-bg` on it's own doesn't really tell me what it does without looking up the additional context. It's also a generic enough name that you could run into trouble with conflicts.